community/kernel/src/main/java/org/neo4j/kernel/IdType.java

```diff
@@ -19,14 +19,16 @@
  */
 package org.neo4j.kernel;
 
+import org.neo4j.unsafe.impl.internal.dragons.FeatureToggles;
+
 /**
  * @deprecated This will be moved to internal packages in the next major release.
  */
 @Deprecated
 public enum IdType
 {
     NODE( 35, false ),
-    RELATIONSHIP( 35, false ),
```

Comment:
problem with old plain `Setting` is that we need to apply it to some `Config` and pass it around, that even more trickier because it's enum, do you really want it to be standard `Setting`?